### PR TITLE
add wiki utilities

### PR DESCRIPTION
**Pull request by alisa911**

### CODE REVIEW

**Comment by alisa911** [comment]
https://github.com/osmandapp/web/issues/545

File: java-tools/OsmAndServerUtilities/src/main/java/net/osmand/server/ServerUtilities.java [hunk 1]
```diff
@@ -37,7 +38,9 @@ public static void main(String[] args) throws Exception {
 			GenerateWebTranslations.generateTranslations(subArgsArray[0], subArgsArray[1]);
 		} else if (utl.equals("parse-styles")) {
 			StyleDataExtractor.parseStylesXml(subArgsArray[0], subArgsArray[1]);
-		} else if (utl.equals("generate-reports")) {
+		} else if (utl.equals("parse-wikidata-licenses")) {
+			WikidataUtilities.parseWikidataLicenses(subArgsArray[0]);
+		}else if (utl.equals("generate-reports")) {
```
Comment (RZR-UA) [inline java-tools/OsmAndServerUtilities/src/main/java/net/osmand/server/ServerUtilities.java:43]: format

File: java-tools/OsmAndServerUtilities/src/main/java/net/osmand/server/utilities/WikidataUtilities.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.util.concurrent.TimeUnit;
+
+public class WikidataUtilities {
+
+	private static final long PARSE_LICENSE_INTERVAL = TimeUnit.DAYS.toMillis(30);
+
+	public static void main(String[] args) {
+		parseWikidataLicenses("../../../osmand/web/map/src/resources/wiki_data_licenses.json");
```
Comment (vshcherb) [inline java-tools/OsmAndServerUtilities/src/main/java/net/osmand/server/utilities/WikidataUtilities.java:19]: location server site

File: java-tools/OsmAndServerUtilities/src/main/java/net/osmand/server/ServerUtilities.java [hunk 3]
```diff
@@ -37,6 +38,8 @@ public static void main(String[] args) throws Exception {
 			GenerateWebTranslations.generateTranslations(subArgsArray[0], subArgsArray[1]);
 		} else if (utl.equals("parse-styles")) {
 			StyleDataExtractor.parseStylesXml(subArgsArray[0], subArgsArray[1]);
+		} else if (utl.equals("parse-wikidata-licenses")) {
```
Comment (vshcherb) [inline java-tools/OsmAndServerUtilities/src/main/java/net/osmand/server/ServerUtilities.java:41]: we don't need to update it regularly I think so we don't need to automate it

**Pull request merged**